app/utils/utils.ts

```diff
@@ -0,0 +1,40 @@
+
+
+export enum LastModified {
+    LAST_THREE_HOURS,
+    TODAY,
+    THIS_WEEK,
+    OLDER
+}
+
+
+export function getTimeFrame(dateTimeString: string): LastModified {
+    try {
+        const dateTime = new Date(dateTimeString);
+        const now = new Date();
+
+        // Check if time is from the last 3 hours
+        const threeHoursAgo = new Date(now.getTime() - (3 * 60 * 60 * 1000));
```

Comment:
I mean, the millisecod multiplication is not pretty but this seems to be a standard way to do it

Reply:
yea gotta calculate the time-diff in milliseconds because of the unix time stamp